cards/src/main/java/com/decathlon/vitamin/compose/cards/VitaminCardSizes.kt

```diff
@@ -0,0 +1,25 @@
+package com.decathlon.vitamin.compose.cards
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.Immutable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+@Immutable
+data class CardSizes(
+    val elevation: Dp
+)
+
+object VitaminCardSizes {
+    @Composable
+    fun medium(
+        elevation: Dp = 1.dp,
+    ): CardSizes = remember(
+        elevation,
```

Comment:
Material Card default radius is 4.dp instead of Vitamin 8.dp
you can add param 
`radius: RoundedCornerShape = VitaminTheme.shapes.radius200`
and use it into Card implementation

Reply:
ok